test/unit/rules/lint-no-obsolete-elements-test.js

```diff
@@ -0,0 +1,258 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-obsolete-elements',
+
+  config: true,
+
+  good: ['<div></div>'],
+
+  bad: [
```

Comment:
This long array of test cases is pretty tedious, and of course duplicates the list of deprecated elements from the rule file, making it more work to update the list later. Instead, you could export the `OBSOLETE_ELEMENTS` array from the rule file, and then import it here to generate the test cases via a `forEach` loop.

Reply:
Ya, something like:

```js
bad: OBSOLETE_ELEMENTS.map(element => {
  return {
    /* generated POJO for this element */
  }
}),
```